packages/core/lib/command-utils.js

```diff
@@ -0,0 +1,209 @@
+const TaskError = require("./errors/taskerror");
+const { bundled, core } = require("../lib/version").info();
+const OS = require("os");
+const analytics = require("../lib/services/analytics");
+const { extractFlags } = require("./utils/utils"); // contains utility methods
+const globalCommandOptions = require("./global-command-options");
+const debugModule = require("debug");
+const debug = debugModule("core:command:run");
+const commands = require("./commands/commands");
+
+const getCommand = (inputStrings, options, noAliases) => {
+  if (inputStrings.length === 0) {
```

Comment:
Is this `inputStrings` here the same as the one in `cli.js`? if so, when the `inputString` length is 0, wouldn't it qualify as `userWantsGeneralHelp` instead of throwing the error?